.circleci/config.yml

```diff
@@ -408,7 +418,9 @@ workflows:
                 - master
                 - /.*(e2e)/
       - test-release:
-          context: GKE
+          context:
+            - GKE
+            - Product-okteto-dev
```

Comment:
This context configures OKTETO_URL and OKTETO_TOKEN pointing to product.okteto.dev